dev/MRTCore/TestApps/winui_class_lib/winui_class_lib.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.ApplicationModel.Resources;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace winui_class_lib
```

Comment:
This is not the naming convention for .NET or WinRT types. It is a C++ naming convention.

I would suggest the namespace be (eg) `SampleLibrary` and the type name be `SampleType` (if indeed the desire is to have a generic sample with no meaningful behaviour other than the "get a string" method.

Reply:
Alright, will fix this in a future, separate change.

Reply:
Will address this in a future, separate change.